src/test/java/de/caritas/cob/agencyservice/api/admin/hallink/AgencyLinksBuilderTest.java

```diff
@@ -28,19 +28,19 @@ void buildAgencyLinks_Should_returnAgencyLinks_When_AgencyIsSet() {
     assertThat(agencyLinks.getSelf(), notNullValue());
     assertThat(agencyLinks.getSelf().getMethod(), is(MethodEnum.GET));
     assertThat(agencyLinks.getSelf().getHref(),
-        is(String.format("/agencyadmin/agencies/%s", agency.getId())));
+        is(String.format("/${openapi.willBeReplaced.base-path}/agencyadmin/agencies/%s", agency.getId())));
```

Comment:
hi @tkuzynow after upgrading openApi those tests failed, so I added this base path. (I couldn't find an other way to fix it)